Gemfile

```diff
@@ -4,6 +4,5 @@ source "https://rubygems.org"
 
 group :development do
   gem "rubocop-discourse"
-  gem "syntax_tree"
-  gem "syntax_tree-disable_ternary"
+  gem "syntax_tree", github: "ruby-syntax-tree/syntax_tree", branch: "main"
```

Comment:
pinning to main since the gem is a dev dependency and isn't version bumped very often, and its fix to get `plugin/disable_auto_ternary` was merged in after the last version.

Reply:
The question is how stable the main branch, but probably it's worth trying.